search-api/search_api/models/corp_party.py

```diff
@@ -314,9 +314,8 @@ def filter_reducer(accumulator, filter_value):
         if sort_type is None:
             results = results.order_by(func.upper(CorpParty.last_nme), CorpParty.corp_num)
         else:
-            sort_field_str = _sort_by_field(sort_type, sort_value)
-            results = results.order_by(eval(sort_field_str))  # pylint: disable=eval-used
```

Comment:
@syin it's preferable to avoid using eval where possible because generating code as strings can be a security hole, and reduced understandability of the code.